MekHQ/src/mekhq/campaign/rating/AbstractUnitRating.java

```diff
@@ -498,28 +488,48 @@ protected void initValues() {
         getLogger().methodEnd(getClass(), "initValues()");
     }
 
-    private void updateBayCount(Dropship ds) {
-        // ToDo Superheavy Bays.
+    void updateBayCount(Entity e) {
+        if ((e instanceof Jumpship) || (e instanceof SmallCraft)) {
+            for (Bay bay : e.getTransportBays()) {
+                if (bay instanceof MechBay) {
+                    setMechBayCount(getMechBayCount() + (int)bay.getCapacity());
+                } else if (bay instanceof BattleArmorBay) {
+                    setBaBayCount(getBaBayCount() + (int)bay.getCapacity());
+                } else if (bay instanceof InfantryBay) {
+                    setInfantryBayCount(getInfantryBayCount() + (int) (bay.getCapacity() / ((InfantryBay) bay).getPlatoonType().getWeight()));
+                } else if (bay instanceof LightVehicleBay) {
+                    setLightVeeBayCount(getLightVeeBayCount() + (int) bay.getCapacity());
+                } else if (bay instanceof HeavyVehicleBay) {
+                    setHeavyVeeBayCount(getHeavyVeeBayCount() + (int) bay.getCapacity());
+                } else if (bay instanceof SuperHeavyVehicleBay) {
+                    setSuperHeavyVeeBayCount(getSuperHeavyVeeBayCount() + (int) bay.getCapacity());
+                } else if (bay instanceof ASFBay) {
+                    setFighterBayCount(getFighterBayCount() + (int) bay.getCapacity());
+                } else if (bay instanceof SmallCraftBay) {
+                    setSmallCraftBayCount(getSmallCraftBayCount() + (int) bay.getCapacity());
+                }
+            }
+        }
+    }
+
+/*    private void updateBayCount(Dropship ds) {
```

Comment:
Do we need to keep the commented out code?

Reply:
Nope, just forgot to remove it